jaclang_jaseci/plugin/jaseci.py

```diff
@@ -0,0 +1,344 @@
+"""Jac Language Features."""
```

Comment:
This will need to be updated too right? Because of most of it will be in jaclang